packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/emrcontainers/start-job-run.ts

```diff
@@ -38,6 +38,16 @@ export interface EmrContainersStartJobRunProps extends sfn.TaskStateBaseProps {
    */
   readonly executionRole?: iam.IRole;
 
+  /**
+   * The execution role arn for the job run.
+   *
+   * Used when both `virtualClusterId` and `executionRoleArn` need to be taken from a JSON input path.
+   * Conflicts with `executionRole` when both provided. `executionRole` takes precedence over `executionRoleArn`.
```

Comment:
If both `executionRole` and `executionRoleArn` are set, please throw an error. This will help developers finding errors when both properties are set and the logic doesn't work as expected.